src/ui/zcl_abapgit_gui_page_repo_view.clas.abap

```diff
@@ -471,9 +491,11 @@ CLASS zcl_abapgit_gui_page_repo_view IMPLEMENTATION.
                          iv_act = |{ zif_abapgit_definitions=>c_action-rfc_compare }?key={ mv_key }|
                          iv_opt = zif_abapgit_html=>c_html_opt-strong ).
       ENDIF.
-      ro_toolbar->add( iv_txt = 'Export <sup>zip</sup>'
-                       iv_act = |{ zif_abapgit_definitions=>c_action-zip_export }?key={ mv_key }|
-                       iv_opt = zif_abapgit_html=>c_html_opt-strong ).
+
+      ro_toolbar->add( iv_txt = 'Export'
+                              iv_opt = zif_abapgit_html=>c_html_opt-strong
+                              io_sub = io_tb_export ).
```

Comment:
what is the relation to the scope of this PR to changing the Export zip functionality?